cdap-docs/reference-manual/source/http-restful-api/lifecycle.rst

```diff
@@ -555,7 +565,7 @@ Example
    * - Description
      - Retrieves the schedules of the Workflow *PurchaseHistoryWorkflow* of the Application *PurchaseHistory*
    * - Returns
-     - ``["WORKFLOW:developer:PurchaseHistory:PurchaseHistoryWorkflow:0:DailySchedule"]``
+     - ``[{"schedule":{"name":"DailySchedule","description":"DailySchedule with crontab 0 4 * * *","cronEntry":"0 4 * * *"},"program":{"programName":"PurchaseHistoryWorkflow","programType":"WORKFLOW"},"properties":{}}]``
```

Comment:
I change it as per suggestions, however the first line is looking bold. Can you please check again?